src/components/entry/Login.vue

```diff
@@ -25,8 +25,10 @@
             <template #entry-main>
                 <section :class="{'entry-form-active': activeOption==='login'}" class="entry-form entry-form-login">
                     <form v-if="!activeGuestLogin" id="login-form" @submit.prevent="loginUser">
-                        <input v-model="user.username" type="text" class="input-field-text" placeholder="Username">
-                        <input v-model="user.password" type="password" class="input-field-text" placeholder="Password">
+                        <input id="userName" v-model="user.username" type="text"
```

Comment:
```suggestion
                        <input id="username" v-model="user.username" type="text"
```
I would keep this all lowercase.

Also as mention in the meeting, if this ends up autocompleting guests users/passwords, I would remove the `autocomplete="on"`.  The browser should still be able to autocomplete them (Firefox already does it for me).